### ISSUE TITLE
how to  render  VertexGroup?

### ISSUE THREAD

**Issue by 442459897** [omegaengine/omegaengine#37]
when I use VertexGroup, it doesn't work. no  engion.  is it a bug?

**Comment by bastianeicher** [comment]
Could you show me a sample of the code that is not working?

A minimal use case could look something like this:
```
var engine = new Engine(form, new EngineConfig());
var scene = new Scene
{
    Positionables = {new VertexGroup(PrimitiveType.TriangleStrip, new []
    {
       new PositionColored(new Vector3(-1,-1,0), 0),
       new PositionColored(new Vector3(0,1,0), 0),
       new PositionColored(new Vector3(1,-1,0), 0)
    }
  }
};
var view = new View(scene, new TrackCamera());
engine.Views.Add(view);
```

**Comment by 442459897** [comment]
# This is my code。

        private void MainForm_Load(object sender, EventArgs e)
        {
            Engine engine = renderPanel.Setup();
            InitializeScene(engine);
            timerRender.Enabled = true;
            engine.FadeIn();
        }

        private void InitializeScene(Engine engine)
        {
            PositionColored[] vertexes ={
                        new PositionColored(new Vector3(-1,-1,0), 0),
                        new PositionColored(new Vector3(0,1,0), 0),
                        new PositionColored(new Vector3(1,-1,0), 0)
                       };
            VertexGroup vg = new VertexGroup(PrimitiveType.TriangleStrip, vertexes);
            var scene = new Scene
            {

                Positionables = { 
                    vg
                }
            };
            _camera = new TrackCamera { VerticalRotation = 20 };
            var view = new View(scene, _camera) { BackgroundColor = Color.CornflowerBlue };
            engine.Views.Add(view);
        }
#  And  this is my error.
************** 异常文本 **************
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 OmegaEngine.Graphics.BufferHelper.GetPoints(VertexBuffer buffer, Int32 vertexCount)
   在 OmegaEngine.Graphics.Renderables.VertexGroup.Initialize(Int16[] indexes)
   在 OmegaEngine.Graphics.Renderables.VertexGroup..ctor(PrimitiveType primitiveType, PositionColored[] vertexes, Int16[] indexes)
   在 OmegaEngineApplication1.MainForm.InitializeScene(Engine engine) 位置 c:\Users\wxz\Documents\Visual Studio 2013\Projects\OmegaEngineDemo\OmegaEngineApplication1\MainForm.cs:行号 41
   在 OmegaEngineApplication1.MainForm.MainForm_Load(Object sender, EventArgs e) 位置 c:\Users\wxz\Documents\Visual Studio 2013\Projects\OmegaEngineDemo\OmegaEngineApplication1\MainForm.cs:行号 28

**Comment by bastianeicher** [comment]
Thanks for finding this. You are right, this is a bug. I'll try get it fixed within the next few days.

**Comment by bastianeicher** [comment]
I have fixed the issue in commit 50b37e2e. A new release of the OmegaEngine libraries is available on NuGet with the version number 1.0.1. Please let me know if it works for you now.
